pandas/core/generic.py

```diff
@@ -378,12 +378,12 @@ def _get_axis_name(cls, axis):
                 pass
         raise ValueError(f"No axis named {axis} for object type {cls.__name__}")
 
-    def _get_axis(self, axis):
+    def _get_axis(self, axis) -> Index:
         name = self._get_axis_name(axis)
         return getattr(self, name)
 
     @classmethod
-    def _get_block_manager_axis(cls, axis):
+    def _get_block_manager_axis(cls, axis) -> int:
```

Comment:
can "axis" be typed as Axis?

Reply:
Yeah, but in in some cases ``axis`` is returned, so that that would require some refactoring inside the function, which I'd like to that seperately. For example I'm prettu sure ``_AXIS_ALIASES`` can be removed.

Reply:
makes sense, thanks